Documentation/Libraries.md

```diff
@@ -0,0 +1,43 @@
+# Library Support
+
+iOS community develops many fantastic libraries to help developers create beautiful, quality applications. Wouldn't it be even better if libraries supported ReactiveCocoa out of the box?
+Certain libraries do not support the framework directly, but wrappers and extensions do exist.
+
+*Below is the list of libraries that support ReactiveCocoa in alphabetical order:*
+
+- [AFNetworking](https://github.com/AFNetworking/AFNetworking)
+  - [AFNetworking-RACExtensions](https://github.com/CodaFi/AFNetworking-RACExtensions)
+  - [AFNetworking-RACRetryExtensions](https://github.com/Legoless/AFNetworking-RACRetryExtensions)
+- [DANReactiveLocationManager](https://github.com/wearebase/DANReactiveLocationManager)
+- [EPSReactiveList](https://github.com/ElectricPeelSoftware/EPSReactiveList)
+- [Expecta](https://github.com/specta/expecta)
+  - [Expecta-ReactiveCocoa](https://github.com/kylef/Expecta-ReactiveCocoa)
+- [JSONModel](https://github.com/icanzilb/JSONModel)
+  - [JSONModel-RACExtensions](https://github.com/Legoless/JSONModel-RACExtensions)
+- [Kiwi](https://github.com/kiwi-bdd/Kiwi)
+  - [ReactiveKiwi](https://github.com/chrisdevereux/ReactiveKiwi)
+- [MEReactiveFoundation](https://github.com/willbur1984/MEReactiveFoundation)
+- [MEReactiveKit](https://github.com/TeamMaestro/MEReactiveKit)
```

Comment:
:heavy_multiplication_x:  Primary not about RAC, though it does use RAC. There are only two classes which expose RAC APIs, neither of which are considerable.